src/app/form-viewer/form-viewer.module.ts

```diff
@@ -0,0 +1,45 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { Routes, RouterModule } from '@angular/router';
+import {
+  MatButtonModule, MatCardModule, MatCheckboxModule, MatIconModule,
+  MatMenuModule, MatSelectModule, MatToolbarModule
+} from '@angular/material';
+import { IonicModule } from '@ionic/angular';
+
+import { FormViewerPage } from './form-viewer.page';
+import {
+  Bootstrap3FrameworkModule,
+  Bootstrap4FrameworkModule,
+  MaterialDesignFrameworkModule,
+  NoFrameworkModule
+} from 'angular6-json-schema-form';
+import { JsonSchemaFormModule } from '../../../node_modules/angular6-json-schema-form/lib/json-schema-form.module';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: FormViewerPage
+  }
+];
+
+@NgModule({
+  imports: [
+    FormsModule,
+    MatButtonModule, MatCardModule, MatCheckboxModule,
+    MatIconModule, MatMenuModule, MatSelectModule, MatToolbarModule,
+    MaterialDesignFrameworkModule,
```

Comment:
There should be a TODO to remove the loading of all the frameworks we don't end up using. I just put them all in for tinkering purposes.